interface/client/appStart.js

```diff
@@ -24,7 +24,7 @@ mistInit = function(){
 
             Tabs.insert({
                 _id: 'browser',
-                url: 'https://ethereum.org',
+                url: 'http://dapps.ethercasts.com/',
```

Comment:
> Will this be officially the Mist first page?
 
No. I just think it's a better placeholder than ethereum.org. We need to create a proper one. I would prefer that it would be exclusive to .eth or swarm uris.